ci-operator/step-registry/openshift/image-based/upgrade/e2e/openshift-image-based-upgrade-e2e-commands.sh

```diff
@@ -17,15 +17,28 @@ host=$(cat ${SHARED_DIR}/ssh_user)
 ssh_host_ip="$host@$instance_ip"
 PULL_SECRET_FILE=$(cat ${SHARED_DIR}/pull_secret_file)
 
-TARGET_VM_NAME=$(cat ${SHARED_DIR}/target_vm_name)
-target_kubeconfig=${remote_workdir}/ib-orchestrate-vm/bip-orchestrate-vm/workdir-${TARGET_VM_NAME}/auth/kubeconfig
+TEST_VM_NAME="unknown"
+case $TEST_CLUSTER in
+  "target")
+    TEST_VM_NAME=$(cat ${SHARED_DIR}/target_vm_name)
+    ;;
+  "seed")
+    TEST_VM_NAME=$(cat ${SHARED_DIR}/seed_vm_name)
+    ;;
+  *)
```

Comment:
both of these can be defined as `TEST_VM_NAME=$(cat ${SHARED_DIR}/${TEST_CLUSTER}_vm_name)`